packages/core/src/services/documents/create.test.ts

```diff
@@ -3,6 +3,9 @@ import { describe, expect, it } from 'vitest'
 
 import { mergeCommit } from '../commits/merge'
 import { createNewDocument } from './create'
+import useTestDatabase from '$core/tests/useTestDatabase'
+
+useTestDatabase()
```

Comment:
I'm not happy with this. I'm not sure the rollback is working on each test. We need to investigate